doc/tut1.rst

```diff
@@ -1047,6 +1047,17 @@ there is a difference between the ``$`` and ``repr`` outputs:
   # --> 3.1400000000000001e+00:3.1400000000000001e+00
 
 
+Type descriptions
+-----------------
+
```

Comment:
It's misleading to have this in tut1. Users coming from Python should instead learn to use "nimsuggest" instead. Types are not runtime values, you don't need to 'echo' them.